ChangeLog.md

```diff
@@ -24,8 +24,10 @@
     ```
     data Proof ext s goal = Proof { pf_extract :: ext, pf_state :: s, pf_unsolvedGoals :: [goal] }
     ```
-  - `try` now uses `commit` instead of `<|>`. This means that we wont
-    generate a useless extract all the time.
+  - Removed `commit`, as it is fundementally broken.
```

Comment:
I think we want commit again after this morning :)

Reply:
It has RETURNED